exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/PageReader.java

```diff
@@ -17,18 +17,24 @@
  */
 package org.apache.drill.exec.store.parquet.columnreaders;
 
+import static org.apache.parquet.column.Encoding.valueOf;
+
+import java.io.EOFException;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
```

Comment:
Please revert changes related to rearranging imports until we don't have determined the correct ordering for all the project.